src/Admin/AdminInterface.php

```diff
@@ -143,7 +143,7 @@ public function getFormBuilder();
     public function getForm();
 
     /**
-     * @throws \RuntimeException if no request is set
+     * @throws \RuntimeException if no request is set NEXT MAJOR: Remove the throw tag
```

Comment:
```suggestion
     * NEXT_MAJOR: Remove the @throws tag.
     *
     * @throws \RuntimeException if no request is set
```

Reply:
using @throws lead to weird behaviour with phpstorm